src/main/java/bowling/domain/controller/BowlingController.java

```diff
@@ -0,0 +1,34 @@
+package bowling.domain.controller;
+
+
+import bowling.domain.Player;
+import bowling.domain.frame.*;
+import bowling.domain.view.InputView;
+import bowling.domain.view.ResultView;
+
+
+public class BowlingController {
+
+    public static void start(Player player) {
+        Frame frame = new NormalFrame(Round.createOf(), null);
+        ResultView.printRound();
+        ResultView.printResult(player.getName());
+
+        while (!frame.isEnd()) {
+            int frameRoundCount = getFrameRoundCount(frame);
```

Comment:
```suggestion
            int frameRoundCount = frame.getFrameRoundCount();
```
```java
//class Frame
public int getFrameRoundCount() {
    Round round = frame.getRound();
    return round.getRound();
}
```
frame 객체의 기능으로 제공해보는건 어떤가요? 🙂